code/game/objects/items/weaponry.dm

```diff
@@ -613,6 +613,8 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 
 /obj/item/melee/baseball_bat/attack(mob/living/target, mob/living/user)
 	. = ..()
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
```

Comment:
```suggestion
	if(HAS_TRAIT(user, TRAIT_PACIFISM) && target != user)
```
... unless?

Reply:
I'm genuinely unconvinced pacifists should be bludgeoning anyone with a baseball bat enough to launch themselves, as that is inherently, and undeniably, a violent action.

If you want to self bat pay for it like everyone else.